From 72b99ef6689ca5628f2a58da50bc9a652115a23e Mon Sep 17 00:00:00 2001
From: Honore Tricot <honorex.tricot@intel.com>
Date: Fri, 15 Feb 2013 14:48:50 +0100
Subject: [PORT FROM MAIN] wifi: Adjust PNO timming according Intel Sleep Policy

BZ: 60560

When PNO is used, Intel SleepPolicy recomendation is
to scan few times during the 5 first minutes then scan every
5 minutes.

Using 19 / 1 / 4 as values for Time / Repeat / Max_Exp_Repeat
permits to get 4 scan before 5 minutes  and then scan
every 5min 4 sec

Feature   : Wlan
Category  : Integration
Purpose   : PV-Enabler
Origin    : Upstream
Upstream-candidate: no

Change-Id: I02e84f67e163a4b2fc932bb0a1af56861c3e7d0d
Orig-Change-Id: Ie1c13c6b7686d36678f31738d3e9fc61873b2c82
Signed-off-by: Honore Tricot <honorex.tricot@intel.com>
---
 src/drivers/android_drv.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/drivers/android_drv.h b/src/drivers/android_drv.h
index 5906527..df9b704 100644
--- a/src/drivers/android_drv.h
+++ b/src/drivers/android_drv.h
@@ -32,18 +32,18 @@
 #define WEXT_PNO_SSID_HEADER_SIZE	2
 #define WEXT_PNO_SCAN_INTERVAL_SECTION	'T'
 #define WEXT_PNO_SCAN_INTERVAL_LENGTH	2
-#define WEXT_PNO_SCAN_INTERVAL		30
+#define WEXT_PNO_SCAN_INTERVAL		19
 /* Scan interval size is scan interval section type + scan interval length
  * above */
 #define WEXT_PNO_SCAN_INTERVAL_SIZE	(1 + WEXT_PNO_SCAN_INTERVAL_LENGTH)
 #define WEXT_PNO_REPEAT_SECTION		'R'
 #define WEXT_PNO_REPEAT_LENGTH		1
-#define WEXT_PNO_REPEAT			4
+#define WEXT_PNO_REPEAT			1
 /* Repeat section size is Repeat section type + Repeat value length above */
 #define WEXT_PNO_REPEAT_SIZE		(1 + WEXT_PNO_REPEAT_LENGTH)
 #define WEXT_PNO_MAX_REPEAT_SECTION	'M'
 #define WEXT_PNO_MAX_REPEAT_LENGTH	1
-#define WEXT_PNO_MAX_REPEAT		3
+#define WEXT_PNO_MAX_REPEAT		4
 /* Max Repeat section size is Max Repeat section type + Max Repeat value length
  * above */
 #define WEXT_PNO_MAX_REPEAT_SIZE	(1 + WEXT_PNO_MAX_REPEAT_LENGTH)
-- 
1.7.4.1

